views/show.tmpl

```diff
@@ -189,6 +189,42 @@
     </div>
   </div>
 </div> <!-- /outer containter -->
+{{if gt (len .Podcasts) 0}}
+  <div class="container-fluid bg-podcasts">
+    <div class="container pt-3 pb-1">
+      <h2>Podcasts</h2>
+      <p>This show comes in podcast form, enjoy this content whenever and wherever you like!</p>
+      <ul class="podcasts">
+      {{range .Podcasts}}
+        <li class="podcast-card box-shadow-1">
+          <img
+            class="podcast-card-image"
+            src="
+          {{if .Photo}}
+            https://ury.org.uk{{.Photo}}
```

Comment:
Also, swap this for {{$.PageContext.FullURL}} :) I think the double quotes on the src attribute would like changing also to single quotes, as to match #244